src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -119,15 +119,33 @@
 
 		<!-- iOS -->
 		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='ios'" Include="android" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='ios'" Include="wasm" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='ios'" Include="macos" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='ios'" Include="skia" />
 		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='ios'" Include="not_ios" />
 		<!-- macOS -->
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='macos'" Include="android" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='macos'" Include="ios" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='macos'" Include="wasm" />
+		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='macos'" Include="skia" />
 		<ExcludeXamlNamespaces Condition="'$(XamarinProjectType)'=='macos'" Include="not_macos" />
```

Comment:
macOS build was failing when processing:

https://github.com/unoplatform/uno/blob/4fb52424e87b930a22171602d12fca59c2d07e07/src/Uno.UI/Microsoft/UI/Xaml/Controls/PullToRefresh/RefreshVisualizer/RefreshVisualizer_themeresources.xaml#L11-L12

It was considered both lines, and a duplicate key error was thrown.
It looks like macOS wasn't excluding `android`, so I fixed `ExcludeXamlNamespaces` for all platforms.

But I don't really understand how things worked previously.

Let's see if the build is going to pass with these changes or not.